samples/storageclass/isilon-replication.yaml

```diff
@@ -0,0 +1,118 @@
+
+  # replication.storage.dell.com/remoteSystem: point to correct remote PowerScale system, this should match whatever clusterName you specified in `isilon-creds` secret
+  # Allowed values: string
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/remoteSystem: "cluster-2"
+
+  # replication.storage.dell.com/rpo: change to any other RPO predefined in a list
+  # Allowed values: "Five_Minutes", "Fifteen_Minutes", "Thirty_Minutes", "One_Hour", "Six_Hours", "Twelve_Hours", "One_Day"
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/rpo: Five_Minutes
+
+  # replication.storage.dell.com/ignoreNamespaces: set to 'true' if you want to ignore namespaces and use one volume group
+  # Allowed values:
+  #   true: ignore namespaces and use one volume group
+  #   false: create separate volume group per namespace
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/ignoreNamespaces: "false"
+
+  # replication.storage.dell.com/volumeGroupPrefix: volume group prefix
+  # Allowed values: string
+  # Optional: true
+  # Default value: None
+  replication.storage.dell.com/volumeGroupPrefix: "csi"
+  # The name of the access zone a volume can be created in
+  # Optional: true
+  # Default value: default value specified in values.yaml
+  # Examples: System, zone1
+  AccessZone: System
+
+  # The base path for the volumes to be created on PowerScale cluster.
+  # Ensure that this path exists on PowerScale cluster.
+  # Allowed values: unix absolute path
+  # Optional: true
+  # Default value: value specified in values.yaml for isiPath
+  # Examples: /ifs/data/csi, /ifs/engineering
+  IsiPath: /ifs/data/csi
+
+  # The permissions for isi volume directory path
+  # This value overrides the isiVolumePathPermissions attribute of corresponding cluster config in secret, if present
+  # Allowed values: valid octal mode number
+  # Default value: "0777"
+  # Examples: "0777", "777", "0755"
+  #IsiVolumePathPermissions: "0777"
+
+  # AccessZone groupnet service IP. Update AzServiceIP if different than endpoint.
+  # Optional: true
+  # Default value: endpoint of the cluster ClusterName
+  #AzServiceIP : 192.168.2.1
+
+  # When a PVC is being created, this parameter determines, when a node mounts the PVC,
+  # whether to add the k8s node to the "Root clients" field or "Clients" field of the NFS export
+  # Allowed values:
+  #   "true": adds k8s node to the "Root clients" field of the NFS export
+  #   "false": adds k8s node to the "Clients" field of the NFS export
+  # Optional: true
+  # Default value: "false"
+  RootClientEnabled: "false"
+
+  # Name of PowerScale cluster, where pv will be provisioned.
+  # This name should match with name of one of the cluster configs in isilon-creds secret.
+  # If this parameter is not specified, then default cluster config in isilon-creds secret
+  # will be considered if available.
+  # Optional: true
+  ClusterName: cluster-1
```

Comment:
We should have this commented, as there is a default which will be used.

Reply:
sure